axum-extra/src/routing/spa.rs

```diff
@@ -201,15 +202,173 @@ where
     }
 }
 
+pub trait SpaRouterExt {
+    fn spa_assets(self, route: &str, path: impl AsRef<Path>) -> Self;
+}
```

Comment:
I don't think an extension trait is right for this. I think its nicer to use `Router::merge` since it accepts `impl Into<Router>`. That means users can build their own types with lots of configuration/builder-methods and have it merged directly into an axum router.

Reply:
Agreed. At first, I thought it'd be nice, but once I built it… it felt not _Axum-ish_ 😅. Will update this later today (if I find the time).